packages/manager/apps/ips/src/pages/listing/index.tsx

```diff
@@ -39,6 +39,14 @@ export default function Listing() {
       label: t('ips_dashboard_guide'),
     },
   ];
+  const navigate = useNavigate();
+
+  const onTabClick = useCallback(
+    (path: string) => {
+      navigate(path);
+    },
+    [navigate],
+  );
```

Comment:
That is equivalent to using navigate directly since you recreate a function every time the function changes.